tests/test_pydbg.py

```diff
@@ -1,10 +1,12 @@
-import os
 import io
+import os
+from contextlib import redirect_stderr; import sys ; sys.path.append(os.getcwd()) ; #XXX
```

Comment:
Can you split this out into 2 lines?
```
from contextlib import redirect_stderr
import sys; sys.path.append(os.getcwd())
```

Reply:
Oh, this was supposed to be removed. Surprised to see it here. Let me drop it.